pygsheets/drive.py

```diff
@@ -48,13 +48,13 @@ class DriveAPIWrapper(object):
     _spreadsheet_mime_type = "application/vnd.google-apps.spreadsheet"
     _folder_mime_type = "application/vnd.google-apps.folder"
 
-    def __init__(self, http, data_path, retries=3, logger=logging.getLogger(__name__)):
+    def __init__(self, http, data_path, retries=3, logger=logging.getLogger(__name__), requestBuilder=None):
```

Comment:
please change `requestBuilder` to `request_builder` for consistency.

Reply:
Done.